lib/spoom/coverage/snapshot.rb

```diff
@@ -22,6 +22,8 @@ class Snapshot < T::Struct
       prop :calls_untyped, Integer, default: 0
       prop :calls_typed, Integer, default: 0
       prop :sigils, T::Hash[String, Integer], default: Hash.new(0)
+      prop :non_generated_methods_with_sig, Integer, default: 0
```

Comment:
I'm not a fan of this name either. What do you think about `methods_with_sig_excluding_rbis`?